src/pages/workspaces/workspace/Data.js

```diff
@@ -501,6 +501,9 @@ const WorkspaceData = _.flow(
 
   const { dataTableVersions, loadDataTableVersions, saveDataTableVersion, deleteDataTableVersion, importDataTableVersion } = useDataTableVersions(workspace)
 
+  const isGoogleWorkspace = !!googleProject
+  const isAzureWorkspace = !isGoogleWorkspace
+
```

Comment:
these are not both strictly necessary, but I think it makes the code below easier to read and understand. I'm open to feedback here.